File: javascript/todo_project/urls.ts
import { Router } from 'express'
import { all, add, update, remove, getById } from '@/todo/routes'

const router = Router()
router.get('/all', all)
router.get('/:id', getById)
router.put('/add', add)
router.patch('/update', update)
router.delete('/delete/:id', remove)
export default router

File: python/todo_project/urls.py

from django.contrib import admin
from django.urls import path
from todo import views #here

urlpatterns = [
    path('admin/', admin.site.urls),
    path('', views.home, name='home'), #here
    path('add/', views.add, name='add'), #here
    path('update/<int:id>/', views.update, name="update"),
    path('delete/<int:id>/', views.delete, name="delete"),

]
